internal/test/wsu/wsu_test.go

```diff
@@ -178,25 +101,25 @@ func TestWSU(t *testing.T) {
 // testFilesCopied tests that the files we attempted to copy to the Windows host, exist on the Windows host
 func testFilesCopied(t *testing.T) {
 	expectedFileList := []string{"kubelet.exe", "worker.ign", "wmcb.exe", "hybrid-overlay.exe", "kube.tar.gz"}
-
 	// Check if each of the files we expect on the Windows host are there
 	for _, filename := range expectedFileList {
 		fullPath := ansibleTempDir + "\\" + filename
 		// This command will write to stdout, only if the file we are looking for does not exist
 		command := fmt.Sprintf("if not exist %s echo fail", fullPath)
 		stdout := new(bytes.Buffer)
-		_, err := winRMClient.Run(command, stdout, os.Stderr)
+
+		_, err := framework.WinrmClient.Run(command, stdout, os.Stderr)
+
```

Comment:
Is this newline intentional?

Reply:
No, when I was doing rebase, I accidentally messed up the newlines. I am reverting those changes. Thanks for catching it